packages/client/src/runtime/core/model/applyModel.ts

```diff
@@ -35,31 +43,36 @@ export function applyModel(client: Client, dmmfModelName: string) {
       // only allow actions that are valid and available for this model
       if (prop in target || typeof prop === 'symbol') return target[prop]
       if (!isValidActionName(client, dmmfModelName, prop)) return undefined
+      const dmmfActionName = getDmmfActionName(prop as ClientModelAction)
 
+      let requestFn = (params: InternalRequestParams) => client._request(params)
```

Comment:
Would it make sense to have a one-liner here to return `requestFn` instead of asserting that it never happens?
```suggestion
      const requestFn = applyClientOnlyWrapper(prop, jsModelName, (p) => client._request(p))
```